tests/System/Commands/MigrateTest.php

```diff
@@ -138,6 +138,14 @@ public function test_runMigration_actuallyCopiesValues()
         $archives = $targetDb->fetchAll('SELECT * FROM ' . self::$fixture->targetDb->prefixTable('archive_blob_2013_01'));
         $this->assertGreaterThanOrEqual(195, count($archives));
         $this->assertLessThanOrEqual(600, count($archives));
+
+        if (!class_exists('Piwik\Plugins\Annotations\AnnotationList')) {
+            $annotationsTable = self::$fixture->targetDb->prefixTable('annotations');
+            $this->assertContains($annotationsTable, $targetDb->listTables(), 'annotations table does not exist');
+
+            $annotationItems = $targetDb->fetchAll('SELECT * FROM ' . $annotationsTable);
+            $this->assertCount(1, $annotationItems);
+        }
```

Comment:
I'm not quite sure if I follow the logic here. L143-L144 establish that there's no annotations table in the target DB and then L146-L147 select from that table? Should these be in if/else instead of a single if statement?

Reply:
@snake14 143-144 check there is the table, the assert message is (I thought) an error message when the assertion fails, not a description of what is asserted.